drivers/include/ad7746.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup   drivers_ad7746 AD7746 Capacitance-to-digital converter driver
+ * @ingroup    drivers_sensors
+ * @ingroup    drivers_saul
+ * @brief      I2C Capacitance-to-digital converter with temperature sensor
+ *
+ * This driver provides @ref drivers_saul capabilities.
+ * @{
+ *
+ * @file
+ * @brief      AD7746 Capacitance-to-digital converter with temperature
+ *             sensor driver
+ *
+ * @author     Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#ifndef AD7746_H
+#define AD7746_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+/**
+ * @brief  AD7746 default address
+ */
+#ifndef AD7746_I2C_ADDRESS
```

Comment:
This can be removed right? You should have the bus address in the device descriptor.